source/physics/include/Gate3GammaAnnihilation.hh

```diff
@@ -25,6 +25,11 @@ public:
   void Initialize();
   void GenerateVertex( G4Event*);
 
+  Double_t calculate_mQED(Double_t mass_e, Double_t w1, Double_t w2, Double_t w3){
+    Double_t mQED = pow((mass_e-w1)/(w2*w3),2) + pow((mass_e-w2)/(w1*w3),2) + pow((mass_e-w3)/(w1*w2),2);
```

Comment:
you can remove the declaration of  the variable and just do
return pow( ....)

by the way you should add some assertion or if before to check if the weigths are non-zero, otherwise you get dividing by zero problem